web/src/features/charts/FuturePrice.tsx

```diff
@@ -190,7 +252,7 @@ function TimeDisplay({ date, granularity }: { date: string; granularity: number
 
   if (isNow(date, granularity)) {
     return (
-      <p className="min-w-[82px] text-sm" data-test-id="now-label">
+      <p className={`min-w-[72px] pl-1 text-sm font-semibold`} data-test-id="now-label">
```

Comment:
You can add a custom size 18 to the tailwind config to avoid using a custom spacing here instead.

72px = 4.5rem = tailwind 18 (don't exist in the default theme).